src/clp_ffi_js/ir/StreamReader.cpp

```diff
@@ -71,7 +72,8 @@ EMSCRIPTEN_BINDINGS(ClpStreamReader) {
             .value("STRUCTURED", clp_ffi_js::ir::StreamType::Structured)
             .value("UNSTRUCTURED", clp_ffi_js::ir::StreamType::Unstructured);
     emscripten::register_type<clp_ffi_js::ir::DecodedResultsTsType>(
-            "Array<{logEventNum: number, logLevel: number, message: string, timestamp: bigint}> "
+            "Array<{logEventNum: number, logLevel: number, message: string, timestamp: bigint, "
+            "utcOffsetMinutes: bigint}> "
```

Comment:
need to make this consistent with the rest of the code. personally i think it's fine to just use "utcOffset" since `minute` is a common unit for that.